MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -85,7 +85,7 @@ class RouteManeuverViewController: UIViewController {
         let stepProgress = routeProgress.currentLegProgress.currentStepProgress
         let distanceRemaining = stepProgress.distanceRemaining
         
-        distance = secondsRemaining > 5 ? distanceRemaining : nil
+        distance = distanceRemaining > 10 ? distanceRemaining : nil
```

Comment:
Confirmed with @bsudekum that it’s OK. This logic exists mainly to avoid ever showing “0 ft”, but location updates are spaced far enough part that we wouldn’t see that in the scenario above.